From b978f460206d63420ad18eb51d4803e1702c6cc4 Mon Sep 17 00:00:00 2001
From: "Jason M. Bills" <jason.m.bills@intel.com>
Date: Mon, 8 May 2023 13:30:44 -0700
Subject: [PATCH] Skip finding tinyxml

tinyxml changed to meson upstream and can no longer be found by CMake.
This updates our downstream build to skip finding tinyxml. Our build
still works because Yocto manages the package.

Upstream, we will work on porting intel-ipmi-oem to meson. Once that is
done, we can delete this patch.

Signed-off-by: Jason M. Bills <jason.m.bills@intel.com>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0e33cec..2e0bcba 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,7 +121,6 @@ target_link_libraries (zinteloemcmds -lchannellayer)
 target_link_libraries (zinteloemcmds ${OPENSSL_CRYPTO_LIBRARY})
 target_link_libraries (zinteloemcmds gpiodcxx)
 
-find_package (tinyxml2 REQUIRED)
 target_link_libraries (zinteloemcmds tinyxml2)
 
 install (TARGETS zinteloemcmds DESTINATION lib/ipmid-providers)
-- 
2.25.1

